code/modules/antagonists/heretic/items/heretic_necks.dm

```diff
@@ -0,0 +1,36 @@
+/obj/item/clothing/neck/heretic_focus
+	name = "Focusing Heart"
+	desc = "A link to the world beyond."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "living_heart"
+	w_class = WEIGHT_CLASS_SMALL
+	clothing_traits = list(TRAIT_ALLOW_HERETIC_CASTING)
```

Comment:
this trait should be given to the thermal and x-ray medallions as well.

there's also some design space for a cloaked medallion that only shows up in the strip menu, not examine messages or your sprite. IIRC, there's a type of heretic robe that does something similar to that, but it'd be nice to have an amulet or something that can do that just for itself without requiring you to wear weird ass robes that cloak EVERYTHING. maybe put it in the same tech node?